Create rds_version_check.guard

avi-harari opened the pull request:
*Issue #, if available:*

*Description of changes:*
Added rds_version_check.guard rule to help validate RDS DB engine versions. This can help with extended support cost among other things.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

avi-harari committed:
Create rds_version_check.guard

aniadev-aws reviewed (approved):
Tested, fine to merge.  Suggested additions to comments (will do another PR)

avi-harari committed:
Edited comments section

Edited comments section to include a better description of variables and expectations
